canvas_modules/common-canvas/src/common-properties/components/field-picker/field-picker.jsx

```diff
@@ -162,9 +162,29 @@ export default class FieldPicker extends React.Component {
 					</div>
 				);
 			}
+			const tooltipId = uuid4() + "-tooltip-fieldpicker";
+			const tooltip = (
+				<div className="properties-tooltips">
+					{String(field.origName)}
+				</div>
+			);
+			let disabled = true;
+			if (field.origName) {
+				disabled = false;
+			}
+			const fieldContentWithTooltip = (<Tooltip
+				id={tooltipId}
+				tip={tooltip}
+				direction="bottom"
+				className="properties-tooltips"
+				disable={disabled}
+				showToolTipIfTruncated
```

Comment:
@nmgokhale The text in the cell should show an ellipsis if truncated, can you see why this isnt working for fieldpicker and expression builder?

This is how it looks like in structures:
<img width="506" alt="image" src="https://user-images.githubusercontent.com/1628100/142256730-466c7179-d119-428f-902c-61b0ade877bb.png">